stdlib/2and3/colorsys.pyi

```diff
@@ -13,3 +12,6 @@ def hsv_to_rgb(h: float, s: float, v: float) -> Tuple[float, float, float]: ...
 ONE_SIXTH = ...  # type: float
 ONE_THIRD = ...  # type: float
 TWO_THIRD = ...  # type: float
+
+# TODO undocumented, should only work with float
+def rgb_to_yiq(r: Union[float, complex], g: Union[float, complex], b: Union[float, complex]) -> Tuple[float, float, float]: ...
```

Comment:
mypy allows you to use the shortcut `complex` for `Union[float, complex]`, I think.